docs/Theming.md

```diff
@@ -486,3 +486,22 @@ You can override the notification duration by setting the `autoHideDuration` pro
 ```
 
 **Tip**: if you use the `showNotification` action, then you can define `autoHideDuration` per message as the third parameter of the `showNotification` action creator.
+
+
+## Loading
+
+For consistency the `Loading` component of `react-admin` is exported. 
```

Comment:
A description of the component's behavior (display a circular progress with optional messages, maybe add ing a link the MUI component too) and utility (display the same loading component as `react-admin` on custom pages for consistency)  would be better.